modules/skeleton/react/stories/visual-testing/stories_Skeleton.tsx

```diff
@@ -0,0 +1,64 @@
+/// <reference path="../../../../../typings.d.ts" />
+import * as React from 'react';
+import {ComponentStatesTable, withSnapshotsEnabled} from '../../../../../utils/storybook';
+import {StaticStates} from '@workday/canvas-kit-labs-react-core';
+import {Skeleton, SkeletonHeader, SkeletonShape, SkeletonText} from '../../index';
+
+export default withSnapshotsEnabled({
+  title: 'Testing|React/Indicators/Skeleton',
+  component: Skeleton,
+});
+
+export const SkeletonStates = () => (
+  <div>
+    <h3>With Custom Line Count</h3>
+    <StaticStates>
```

Comment:
`StaticState` isn't used anywhere. This only does something if `className` is used as a prop and only for pseudo states like `.focus` or `.hover`.

I agree with @lychyi where the table isn't necessary to show all component states - especially in this example where there isn't any matrix. But it does show nice text descriptions of what we're looking at. I think visual tests, like our functional tests, should have context easily available. The way the text is laid out helps describe what's in the visual snapshot.

The confusing part here is which skeleton types are being used.

First table is `SkeletonText`, but the label says "With Custom Line Count" even though the "Default" has no custom line count. Your variant would be your line count of 4.

Second table is `SkeletonShape`. with a few different widths, heights, and border radii.

Third table has `SkeletonShape`, `SkeletonHeader`, and `SkeletonText`. I can see by the story that `SkeletonHeader` is not obvious and makes me wonder if there are different header types (heights). I'd like to see side-by-side comparisons of skeletons to actual content. A common thing I see people get wrong with skeleton loading is that the "skeleton" isn't actually a skeleton of the layout - so layout shifts. Sometimes dramatically which defeats the entire purpose of skeleton loaders. We should have stories that show what they look like side-by-side so we can see that they line up correctly. We should have some stories that show loading at different loading times (best practices of _when_ to show them) with the ability to reset.

Skeleton loaders are abused so many times and we don't help the situation since we don't provide any good examples.